homework_26_11_1/src/Laptop.java

```diff
@@ -0,0 +1,65 @@
+public class Laptop {
+    String brand;
+    boolean isTurnedOn;
+    String welcomePhrase;
+    int butteryAmount;
+
+    public Laptop (String brand, boolean isTurnedOn, String welcomePhrase, int butteryAmount) {
+        this.brand=brand;
+        this.isTurnedOn=isTurnedOn;
+        this.welcomePhrase=welcomePhrase;
+        this.butteryAmount=butteryAmount;
+    }
+
+    public String convertStatus(boolean currentStatus) {
+        if (currentStatus) {
+            return "ON";
+        }
+        else {
+            return "OFF";
+        }
+    }
+    public String getLaptopBrand () {
+        return brand;
+    }
+
+    public boolean getIsTurnedOn() {
+        return isTurnedOn;
+    }
+
+    public boolean setIsTurnedOn (boolean isTurned) {
+        this.isTurnedOn = isTurned;
+        return isTurnedOn;
+    }
+
+    public boolean TurnOnOff() {
+        boolean currentStatus = getIsTurnedOn();
+        return setIsTurnedOn(!currentStatus);
+    }
+
+    public void getWelcomePhrase () {
+        System.out.println(welcomePhrase);
+    }
+
+    public int getButteryAmount () {
+        return butteryAmount;
+    }
+    
+    public void setButtery (){
+        int currentAmount = getButteryAmount();
+        butteryAmount -=Math.min(10,currentAmount);
```

Comment:
what do you want to achieve by Math.min(10,currentAmount)?

Reply:
the battery charge cannot be lower than 0. If we lower the battery charge with each step, for example, from the initial 55, in the last stage the battery charge goes to -5
To avoid this, I used Math.min